common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -105,6 +106,22 @@ def has_course_ended(end_date):
     return datetime.now(UTC()) > end_date if end_date is not None else False
 
 
+def course_starts_within(start_date, look_ahead_days):
+    """
+    Given a course's start datetime and look ahead days, returns whether
+        (a) arguments are not None, and
+        (b) course starts with in days specified by 'look_ahead_days'.
+
+    Arguments:
+        start_date (datetime): The start datetime of the course in question.
+        look_ahead_days (int): number of days to see in future for course start date.
+    """
+    if not isinstance(start_date, datetime) or not isinstance(look_ahead_days, int):
```

Comment:
I don't think we should check the type of parameters here since those are not user inputs.